kiva/quartz/ABCGI.pyx

```diff
@@ -1791,10 +1790,10 @@ cdef class CGImageFile(CGImage):
         self.bmp_array = c_numpy.PyArray_SimpleNew(3, &(dims[0]), c_numpy.NPY_UBYTE)
 
         data = self.bmp_array.data
-        s = piltostring(img)
-        py_data = PyBytes_AsString(s)
+        bs = img.tobytes()
+        py_data = PyBytes_AsString(bs)
 
-        memcpy(<void*>data, <void*>py_data, len(s))
+        memcpy(<void*>data, <void*>py_data, len(bs))
```

Comment:
It might be better not to touch this code even though there is an array object involved.